drivers/scheduler/k8s/specs/pgbench/px-pgbench-storage.yaml

```diff
@@ -23,7 +23,7 @@ spec:
     - ReadWriteOnce
   resources:
     requests:
-      storage: 5Gi
```

Comment:
are these changes related to the subject of the PR? Maybe we can create separate one for that?

Reply:
yes @stgleb , we need this change because pgbench app itself is writing 10GB data [here](https://github.com/portworx/torpedo/blob/master/drivers/scheduler/k8s/specs/pgbench/px-pgbench-app.yaml#L52), so need to increase size of PVC.